assets/stylesheets/sections/gutenberg-editor.scss

```diff
@@ -1,34 +1,3 @@
-@import '../shared/colors';
-@import '../shared/functions/z-index';
-@import '../shared/mixins/breakpoints';
-@import '../shared/mixins/clear-fix';
-@import '../shared/mixins/clear-text';
-@import '../shared/mixins/long-content-fade';
-@import '../shared/mixins/placeholder';
-@import '../shared/typography';
```

Comment:
Did we not need any of the mixins in this section?

Reply:
The `./media` import imports all the shared mixins, too, so nothing gets lost. I hope all the media styles will be migrated to webpack CSS soon. #28790 implements a big part of the task.